src/main/java/net/datafaker/providers/base/Aviation.java

```diff
@@ -28,6 +32,14 @@ public String warplane() {
         return resolve("aviation.aircraft.warplane");
     }
 
+    public String general() {
```

Comment:
Should these just be added to “airplane”?

If not please add some JavaDoc to both that clarifies the distinction

Reply:
Added JavaDoc.